popup.html

```diff
@@ -7,21 +7,23 @@
   </head>
   <body class="popup-body">
     <div class="content-flex">
-      <div class="left-span">Sponsored&nbsp;<b>flagged</b>:</div>
-      <b
-        ><div class="right-span circle" id="sponsoredCount">
-          loading count...
-        </div></b
-      >
+      <div class="left-span">
+        <span>Sponsored <b>flagged</b>:</span>
+      </div>
+      <div class="right-span circle">
+        <p id="sponsoredCount">loading count...</p>
+      </div>
```

Comment:
Since we're talking about semantics. The [dl, dt, dd](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) group of elements, might be better suited for displaying metadata.

Reply:
Could you elaborate or provide some documentation?

Reply:
@QZera interesting!
Reading the docs for those html tags I have some difficulties understating what metadata could be. So far, I could say that metadata could be details of a term in a page (i.e. usage of h1, h2, etc.).

In the context of this repo, do you think that the counter express metadata?

Reply:
@gtopsis 
In this case, I would consider the counter as metadata for the extension, yes. But I didn't mean to imply that these elements should only be used for metadata. I think they're a good way to represent name-value pairs like (in this case) statistics.

e.g.
```html
<dl>
  <dt>Sponsored flagged:</dt>
  <dd>{count}</dd>
</dl>
```

Reply:
@QZera sounds good to me. Changed the code accordingly.
Let's see what the author thinks.